### PR TITLE
fixes postgres image issue

### LINKED ISSUE #101: Postgres:latest docker image isn't working locally for me
The issue talks about the error I am getting when I run docker-compose up to which contains three services.

- The DB which is Postgres: latest
- the web
- The API

The Postgres: latest image is not working for me locally

below is the error.

postgres

### PR DESCRIPTION
**Pull request by phawazzzy**
closes #101 
Over couples of weeks now, I have been having issues with starting the app locally due to the Postgres image issue.

This PR contains a fix that worked locally for me. 

what I did was to change the Postgres image I was using to Postgres:alpine image.

I will you all to pull this branch to see if the new postgres image used will work for you locally

Below is a screenshot of the terminal which shows the working state when I ran with docker
Screenshot 2021-04-30 at 1 02 44 PM

Screenshot 2021-04-30 at 1 02 53 PM

I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by adefemi171** [comment]
recheck

**Review by adefemi171 — APPROVED** [review]

**Comment by nextwebb** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by nextwebb — APPROVED** [review]

**Comment by simplytunde** [comment]
what issue is this linked to?

File: .gitignore [hunk 1]
```diff
@@ -28,4 +28,5 @@
 **yarn-error.log*
 **yarn.lock*
 
-.idea/
\ No newline at end of file
+.idea/
+.env
```
Comment (adefemi171) [inline .gitignore:32]: @phawazzzy I think we are leaving .env not hidden intentionally, so this should not be in the file

File: .env [hunk 2]
```diff
@@ -3,8 +3,6 @@ DB_PORT="5432"
 DB_USER="root"
 DB_PASSWORD="password"
 DB_NAME="boardly"
-
-SERVER_HOST="localhost"
+SERVER_HOST="127.0.0.1"
```
Comment (adefemi171) [inline .env:6]: Also is there a reason why you are not using `localhost `here instead?

File: .gitignore (hunk 1 above)
Comment (adefemi171) [reply .gitignore:32]: Yes

File: .env (hunk 2 above)
Comment (phawazzzy) [reply .env:6]: no particular reason, just me when i was debugging 
i will change it to the IP address now

**Comment by phawazzzy** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by nextwebb — DISMISSED** [review]

**Review by adefemi171 — APPROVED** [review]

**Review by nextwebb — APPROVED** [review]

**Pull request merged**